### ISSUE TITLE
Question about the label image

### ISSUE THREAD

**Issue by qmy612** [vqnhat/DSN-Binarization#1]
In your data set, the label image is 0/255, while I fine-tune on my own data set(label is also 0/255), but the result is not very good, even worse than result on your pre-trained model, because the word has holes and brokes, and the edge of page is detected but this part is useless.
I also try label (0/1) image to fine-tune, but the loss is 0.
I feel puzzled about the result, if you have any idea, please tell me.

**Comment by vqnhat** [comment]
Sorry for my late reply.

Could you let me know some information about the training?

1. Have you tried to train with different learning rate values? In my experience, the learning rate for fine-tuning should be smaller.

2. Which pre-trained model did you use for the initialization?

3. The size of training image patches is also important. Could you let me know the size of the image patches that are used for training?

**Comment by justinner** [comment]
Hi,I can not open the web of dataset and code due to my google VPN,could you help me?

**Comment by vqnhat** [comment]
> Hi,I can not open the web of dataset and code due to my google VPN,could you help me?

Hi, 

Please tell me the storage service that you can access to. I will upload the data to that service so that you can get it.

p/s: you can try these links for the code and data
https://unioulu-my.sharepoint.com/:u:/g/personal/qvo_univ_yo_oulu_fi/EaiGwrHoMpZPq6qilRp4YbcBerDNJU49pAUh-5qomyw5nQ?e=VruAbK

https://unioulu-my.sharepoint.com/:u:/g/personal/qvo_univ_yo_oulu_fi/ETDZymJwN9tHnKNUKui1j3sBZMmRpuops1sSp7-LyA6XKw?e=0TSSeG

**Comment by justinner** [comment]
Thank you very much,I can open and download now,I am really grateful of you .

**Comment by Thomnt2306** [comment]
hi,i'm trying to training on the data you provide,but it's loss too big,i'm very bewildered,hope you contribute,thaks
![Screenshot from 2019-03-30 11-35-17](https://user-images.githubusercontent.com/49086407/55271487-eaae8000-52df-11e9-9b11-58d027979b5c.png)

**Comment by vqnhat** [comment]
@Thomnt2306 
hi, it seems that the network will need more time for the convergence. The training usually requires over 400000 iterations.

**Comment by Thomnt2306** [comment]
Thank you very much for your feedback, I would like to talk more about the problem I am having, I am running training on the CPU and the speed is very slow, when I use base_lr: le-07 like yours then loss=nan,when down le-08 get the results above or loss=0, hope you can help,thanks.

**Comment by vqnhat** [comment]
Hi, the training should be run on the GPU for increasing the speed. Regarding the learning rate, it might be because of the differences in the environment. You can test and select the learning rate that helps reducing the loss value during training (in your case, it might be le-08).

**Comment by Thomnt2306** [comment]
thanks very much,can you tell me loss mean your?

**Comment by Thomnt2306** [comment]
thanks very much,Can you tell me the value loss mean and accuracy your?
